x-pack/plugins/lens/public/editor_frame_service/editor_frame/config_panel/_layer_panel.scss

```diff
@@ -27,6 +27,14 @@
   overflow: hidden;
 }
 
+.lnsLayerPanel__dimension-hidden {
```

Comment:
nitpick: in other places in lens we use BEM notation with a format `block__element--modifier` so you might want to change it to `.lnsLayerPanel__dimension--hidden` and `lnsLayerPanel__dimension--replace`

Reply:
BEM also indicates that you use a single dash for specific **states**. Like `block__element-isDisabled`. The single dash is technically correct here, but it's better to indicate state by adding the `is` in front like: `lnsLayerPanel__dimension-isHidden`